model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt

```diff
@@ -83,12 +84,20 @@ data class ResolvedLicenseInfo(
      * in any of the configured [LicenseFilePatterns] matched against the root path of the package (or project).
      */
     fun mainLicense(): SpdxExpression? {
-        val licenseMatcher = PathLicenseMatcher(LicenseFilePatterns.getInstance())
-        val licensePaths = flatMap { resolvedLicense ->
+        val licenseFilePatterns = LicenseFilePatterns.getInstance()
```

Comment:
commit-message: If this fixes this serious performance issue, I believe this should be made more prominent. 

Furthermore, could you add some details, why exactly previously it was so slow?

Reply:
For these topics, I'd refer to the original fix in ebc6fe0c745067f1f504dd244b574a15ef9b8b34.